lib/workers/pr/body/__snapshots__/controls.spec.ts.snap

```diff
@@ -5,7 +5,7 @@ exports[`workers/pr/body/controls getControls when the branch is  modified has t
 
 ---
 
- - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check this box. ⚠️ **Warning**: custom changes will be lost.
+ - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check this box. ⚠ **Warning**: custom changes will be lost.
```

Comment:
Old:
![image](https://user-images.githubusercontent.com/1239644/116581848-cc376a80-a925-11eb-91f1-f39e7739ba8d.png)

New:
![image](https://user-images.githubusercontent.com/1239644/116581888-d9545980-a925-11eb-9e2e-61f66a69d73f.png)

Is that critical?

Reply:
OMG, it will cause lots of PRs being rebased in runtime

Reply:
I think it's ok to update all open pr's, as it's a onetime change and normally causes no notifications.

@rarkins I'm otherwise LGTM with this pr